app/src/main/java/com/fstyle/androidtrainning/screen/detailsmovie/DetailsMovieActivity.java

```diff
@@ -12,9 +12,18 @@
 import android.widget.ImageView;
 import android.widget.RelativeLayout;
 import android.widget.TextView;
+import android.widget.Toast;
 import com.bumptech.glide.Glide;
 import com.fstyle.androidtrainning.MainApplication;
 import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.data.local.DeleteMovieFromDatabase;
+import com.fstyle.androidtrainning.data.local.GetMovieFromDatabase;
+import com.fstyle.androidtrainning.data.local.InsertMovieToDatabase;
+import com.fstyle.androidtrainning.data.local.MovieDatabase;
+import com.fstyle.androidtrainning.data.local.OnDeleteDataListener;
+import com.fstyle.androidtrainning.data.local.OnInsertDataListener;
+import com.fstyle.androidtrainning.data.local.OnSelectDataListener;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.data.local.OnSelectDataListener. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports